airmozilla/main/templates/main/event.html

```diff
@@ -177,7 +177,6 @@ <h1 class="entry-title">
               {% for chapter in chapters %}
               <li>
                 <a href="#@{{ show_duration_compact(chapter.timestamp) }}" data-ts="{{ chapter.timestamp }}">
```

Comment:
Add a title attribute to this tag. E.g `title="{{ show_duration(chapter.timestamp, include_seconds=True) }}"`